task/buildah/0.2/buildah.yaml

```diff
@@ -487,16 +492,19 @@ spec:
       mountPath: /mnt/trusted-ca
       readOnly: true
     workingDir: $(workspaces.source.path)
-
+  - name: icm
+    image: quay.io/konflux-ci/icm-injection-scripts:latest@sha256:462980e94ba689b5f56c3d5dfb3358cd8c685300daf65a71532f11898935e7f1
```

Comment:
Two questions:

- Where is the source code to this thing?
- Why is the container image missing standard metadata to describe its sources? (Its `"org.opencontainers.image.url": "https://fedoraproject.org/"` is apparently inherited from Fedora 40)

Reply:
answering the first part of my own question by searching github, looks like it's in https://github.com/konflux-ci/build-tasks-dockerfiles/tree/main/icm-injection-scripts

Reply:
The answer to the second part is that the labels are not overwritten in the Containerfile: https://github.com/konflux-ci/build-tasks-dockerfiles/blob/main/icm-injection-scripts/Containerfile#L7

The fact that they persisted to this image means that they were also not overwritten in the parent images, i.e. https://github.com/konflux-ci/buildah-container/blob/main/Containerfile.task and https://github.com/konflux-ci/buildah-container/blob/main/Containerfile.buildah